101-javaee-like-getting-started/src/test/java/io/quarkus/qe/core/JavaEELikeHealthCheckTest.java

```diff
@@ -16,12 +22,29 @@ public void testHealthEndpoint() {
                 .then()
                 .statusCode(200)
                 .body("status", is("UP"),
-                        "checks.name", containsInAnyOrder("liveness", "readiness", "greeting", "Database connections health check"),
```

Comment:
I would like to still see the dummy `greeting` health check

Reply:
Or you intentionally repurposed the `greeting` check for newer stuff?

Reply:
I only thought it was a good idea to cover the HealthGroups. But I can revert this test case and change the old `@Health` by `@Liveness` if you prefer so. 
What do you think?

Reply:
I've changed the `@Health` to `@Readiness` and added a new test case to cover `@HealthGroup`.